contracts/core/AuxiliaryBlockStore.sol

```diff
@@ -236,7 +276,7 @@ contract AuxiliaryBlockStore is BlockStore {
      *
      * @return transitionHash_ Hash of transition object at the checkpoint.
      */
-    function  auxiliaryTransitionHashAtBlock(
+    function transitionHashAtBlock(
```

Comment:
👍

Reply:
True. Transition object should probably be updated as well.

Reply:
That's not possible, because solidity doesn't allow overriding methods with different returns. It's a compilation error.

Reply:
Created `OriginTransitionObjectInterface` and `AuxiliaryTransitionObjectInterface`.